src/main/java/com/tradeshfit/model/dao/SpringMessagesDAO.java

```diff
@@ -0,0 +1,37 @@
+package com.tradeshfit.model.dao;
+
+import com.tradeshfit.model.dao.mapper.MessageMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import javax.sql.DataSource;
+import java.util.Date;
+import java.util.List;
+
+public class SpringMessagesDAO implements MessagesDAO {
```

Comment:
I'd say this is more of a `JdbcMessagesDAO` than a Spring implementation, but I suppose both work.